webcompat/views.py

```diff
@@ -5,6 +5,7 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 import json
+import requests
```

Comment:
this should come after, separate from the standard library. It is an imported module.